src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const newStyles = sourceString.trim();
+  const stylesArray = newStyles.split(';');
+  const arr = [];
```

Comment:
[CODE STYLE] - Avoid variable namings like `arr`, it will decrease code readability. Use more descriptive names that reflect the content of the variable, such as `styleDeclarations`.